pyo3-macros-backend/src/pyclass.rs

```diff
@@ -504,10 +504,10 @@ impl<'a> PyClassComplexEnum<'a> {
                 let variant = match &variant.fields {
                     Fields::Unit => {
                         bail_spanned!(variant.span() => format!(
-                        "Unit variant `{ident}` is not yet supported in a complex enum\n\
-                        = help: change to a struct variant with no fields: `{ident} {{ }}`\n\
-                        = note: the enum is complex because of non-unit variant `{witness}`",
-                        ident=ident, witness=witness))
+                            "Unit variant `{ident}` is not yet supported in a complex enum\n\
+                            = help: change to a struct variant with no fields: `{ident} {{ }}`\n\
+                            = note: the enum is complex because of non-unit variant `{witness}`",
```

Comment:
Maybe a tuple variant with no fields now? Or is struct-like still better do you think? Either way, let's make the doc suggestion I gave above match this.